2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2047 @@
+develop, test, validate, and benchmark features in a continuous
+integration cycle. The challenge in this context is the persistent
+evolution of the hardware landscape. Hence, for creating an environment
+promoting sustainable software development, it is important to provide
+access to a wide hardware portfolio and to support a development cycle
+based on continuous integration.
+
+Expertise in sustainable research software development is a scarce
+resource, and training is heavily needed as one way of building up more
+expertise. However, while integrating interdisciplinary software
+engineering courses into the education curriculum can build up basic
+skills, some expertise is domain-specific and requires
+inter-institutional training activities. Furthermore, there exist no
+financial incentives for creating software-specific documentation and
+tutorials nor to provide other forms of support.
+
+While the creation of research software communities is one of the major
+assets in sustaining research software technology, promoting this
+process requires the installation of new funding instruments.
+Traditionally, research grants are limited to rather short time frames
+and support personnel, material, hardware, and to a limited degree also
+travel and research visits. Creating a research software community
+however requires funding for community and training events as well as
+“virtual hardware” such as webspace, versioning systems, task-managing
+systems, and compute cycles. These demands can hardly be met without
+third-party funding (Kuchinke et al. 2016; Chang, Mills, and Newhouse
+2007; Aartsen et al. 2018; Gabella, Durinx, and Appel 2018).
+
+Recommendation: Creation of Adequate Funding Schemes
+----------------------------------------------------
+
+Funding is a crucial factor for sustaining research software. Currently
+available sources and instruments are not adequately shaped for the
+challenges and solutions outlined above. We recommend actions on the
+individual, organizational, and national level.
+
+Existing project-focused funding instruments on the local, national, and
+international level need to be complemented with funding instruments
+specifically designed for research software development and sustained
+research software maintenance to make research software a first class
+citizen in the research landscape. For example, software projects
+enhancing research and fulfilling the sustainability criteria detailed
+above may be entitled for sustained funding as long as they
+live up to the standards and remain a central component of the research
+landscape.
+
+Computing centers and supercomputing facilities for research need to
+receive earmarked resources for the support of sustainable software
+development. This funding is necessary to provide continuous integration
+services, a hardware portfolio for development, testing and benchmarking
+software, as well as personnel for traini
```

Comment:
Not quite true ("lacking"/"globally accessible", cf. Invenio, Zenodo, Software Heritage). Perhaps focus more on usability of platforms (e.g., GH-Zenodo bridge), structural issues (push/pull methods for Software Heritage), awareness?

Reply:
Transferred to Overleaf.